app/src/main/java/com/teamltt/carcare/database/contract/ReminderContract.java

```diff
@@ -0,0 +1,140 @@
+        String[] selectionArgs = {Long.toString(vehicleId)};
+
+        return db.query(table, columns, selection, selectionArgs, null, null, null);
+    }
+
+    /**
+     * @param db the writabtle SQLiteDatabase
+     * @param featureId a long, the id of the feature being checked, 0 if feature is date
+     * @param comparison an int, 0 if <, 1 if =, 2 if >
+     * @param value an int, the value used in the comparison
+     * @param date a String
+     * @return the new vehicle's primary key
+     */
+    public static long insert(SQLiteDatabase db, long vehicleId, String name, int featureId, int comparison, int value, String date) {
+        ContentValues values = new ContentValues();
+        values.put(ReminderEntry.COLUMN_NAME_VEHICLE_ID, vehicleId);
+        values.put(ReminderEntry.COLUMN_NAME_NAME, name);
+        values.put(ReminderEntry.COLUMN_NAME_FEATURE_ID, featureId);
+        values.put(ReminderEntry.COLUMN_NAME_COMPARISON, comparison);
+        values.put(ReminderEntry.COLUMN_NAME_VALUE, value);
+        values.put(ReminderEntry.COLUMN_NAME_DATE, date);
+        return db.insert(ReminderEntry.TABLE_NAME, null, values);
+    }
+
+    /**
+     * @param db the writable SQLiteDatabase
+     * @param reminderId a long
+     * @param featureId a long, the id of the feature being checked, 0 if feature is date
+     * @param comparison an int, 0 if <, 1 if =, 2 if >
+     * @param value an int, the value used in the comparison
+     * @param date a String
+     * @return the number of rows affected
+     */
+    public static int update(SQLiteDatabase db, long reminderId, long vehicleId, String name, int featureId, int comparison, int value, String date) {
+        ContentValues values = new ContentValues();
+        values.put(ReminderEntry.COLUMN_NAME_VEHICLE_ID, vehicleId);
+        values.put(ReminderEntry.COLUMN_NAME_NAME, name);
+        values.put(ReminderEntry.COLUMN_NAME_FEATURE_ID, featureId);
+        values.put(ReminderEntry.COLUMN_NAME_COMPARISON, comparison);
+        values.put(ReminderEntry.COLUMN_NAME_VALUE, value);
+        values.put(ReminderEntry.COLUMN_NAME_DATE, date);
+        String whereClause = ReminderEntry.COLUMN_NAME_ID + " = ?";
+        String[] whereArgs = {Long.toString(reminderId)};
+        return db.update(ReminderEntry.TABLE_NAME, values, whereClause, whereArgs);
+    }
+
+    public static int delete(SQLiteDatabase db, long reminderId) {
+        String whereClause = ReminderEntry.COLUMN_NAME_ID + " = ?";
+        String[] whereArgs = {Long.toString(reminderId)};
+        return db.delete(ReminderEntry.TABLE_NAME, whereClause, whereArgs);
+    }
+
+    // HACK: private to prevent someone from accidentally instantiating a contract
+    private ReminderContract() {
+    }
+
+    public static class ReminderEntry {
+        public static final String TABLE_NAME = "reminders";
+        public static final String COLUMN_NAME_VEHICLE_ID = "vehicle_id";
```

Comment:
Is this the same `vehicle_id` as from the vehicles table? If so, reference the `VehicleContract.VehicleEntry.COLUMN_NAME_ID` String.